extensions/amp-video-service/0.1/amp-video-service.js

```diff
@@ -52,16 +51,21 @@ const TAG = 'amp-video-service';
  * If you need to add methods to this class that are public to components,
  * it's most likely that you'll want to implement them here and set wrappers for
  * them in the runtime-level service class.
- *
- * @visibleForTesting
```

Comment:
I originally thought this would be `visibleForTesting` but other modules need access to the type.